h/templates/activity/search.html.jinja2

```diff
@@ -348,18 +348,26 @@
     {% endif %}
 
     {% if timeframes %}
-      <ol class="search-result-list {%- if more_info %} search-result-hide-on-small-screens{% endif %}">
-        {% for timeframe in timeframes %}
+
+      <div class="search-results">
```

Comment:
I will add a comment in search.scss. I spent at least 20 mins solely thinking through how to rename it without refactoring all of the naming. Given how people interact with CSS updating and debugging, the current naming and a comment should suffice.